shell.nix

```diff
@@ -1,13 +1,14 @@
-with import <nixpkgs> {};
-
-pkgs.mkShell {
-  name = "area13";
-
-  buildInputs = [
-    terraform
-    tfsec
-    terrascan
-    ripgrep
-    bat
-  ];
-}
+# See https://nixos.wiki/wiki/Flakes#Using_flakes_project_from_a_legacy_Nix
+(import
+  (
+    let
+      lock = builtins.fromJSON (builtins.readFile ./flake.lock);
+    in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  {
+    src = ./.;
+  }).shellNix
```

Comment:
if you want to changes shell, please update in flake.nix at attribute devShell that run same function as before in shell.nix (mkShell).